spectral_fitting/parallel_spectral_fitting.ipynb

```diff
@@ -0,0 +1,1170 @@
+     },
+     "metadata": {
+      "needs_background": "light"
+     }
+    }
+   ],
+   "source": [
+    "masked_moment0 = masked_cube.moment0()\n",
+    "plt.imshow(masked_moment0.value, origin='lower', cmap='inferno')\n",
+    "cbar = plt.colorbar()\n",
+    "cbar.set_label('Integrated Intensity (K km/s)')\n"
+   ]
+  },
+  {
+   "source": [
+    "The regions kept after masking is what we expect: 2 Giant Molecular Clouds (GMCs) in the field-of-view and two fainter regions that have been cut-off in the tutorial data.\n",
```

Comment:
This sounds like four regions not three - maybe make it clear the 2 GMCs count as one above?